commands/intakecommands.py

```diff
@@ -0,0 +1,89 @@
+from commands2 import ParallelCommandGroup
```

Comment:
please put these commands into robotcontainer if they don't already exist there, buttons should already be in OI